### PR TITLE
created tests for route bookSuggestion model

### PR DESCRIPTION
**Pull request by LYAbdala**
# **Sumary**

-Created test for routes of BookSuggestion model.

# **Trello Card**

https://trello.com/c/Tr3Cu9Ny/31-crear-test-para-endpoint-de-creacion-de-un-booksuggestion

### CODE REVIEW

File: app/controllers/api/v1/book_suggestions_controller.rb [hunk 1]
```diff
@@ -9,7 +9,7 @@ def create
       end
 
       def index
-        render json: policy_scope(BookSuggestion), status: :ok
+        render json: BookSuggestion.all, status: :ok
```
Comment (Dinuuu) [inline app/controllers/api/v1/book_suggestions_controller.rb]: Why this change ?

Comment (LYAbdala) [inline app/controllers/api/v1/book_suggestions_controller.rb]: Because I was looking for the source of a mistake. Forget to undo that change.
